### PR TITLE
feat(schematics): Customizing naming of state folder

### LINKED ISSUE #94: Provide a way to speicify ngrx folder instead of '+state'
I am not really sure if I should file this a feature request or not. Being mostly a python shop, we are set in our ways of naming folder structures, which usually begins with '_', so our OCD is preventing us from using this awesome package. 

Does it make sense?

### PR DESCRIPTION
**Pull request by ankemp**
Adding ability to customize the name of the folder generated with `ng g ngrx [name]`.
Adds `--directory=directoryName` flag.
Usage example:
```bash
$ ng g ngrx myState --directory=myState --module=libs/myFeature/myFeature.module.ts
```

Fixes #94

### CODE REVIEW

File: packages/schematics/src/collection/ngrx/index.ts [hunk 1]
```diff
@@ -27,6 +27,10 @@ function addImportsToModule(name: string, options: Schema): Rule {
       return host;
     }
 
+    if (!options.directory) {
+      options.directory = '+state';
```
Comment (vsavkin) [inline packages/schematics/src/collection/ngrx/index.ts]: is this required? I thought `"default": "+state",` should do the same?

**Comment by vsavkin** [comment]
One comment to address. Other than that it looks good

File: packages/schematics/src/collection/ngrx/index.ts (hunk 1 above)
Comment (ankemp) [reply packages/schematics/src/collection/ngrx/index.ts]: Didn't realize the `schema.json` actually handled that. Should've tested it. My b.

**Comment by vsavkin** [comment]
Thank you! Merging it.

**Pull request merged**